user-stories/US-07-Data-rights-request-capture-service.md

```diff
@@ -0,0 +1,127 @@
+
+### US-07-1-04
+
+**As an** internet user (a submitter or a data subejct), when creating a Data rights request (Global data rights request capture service)
+
+**I want to** have a proposition of addresses related to the organization I want to address my Data rights request to, if the address is known
+
+**because** it saves me time and it is convenient
+
+### US-07-1-05
+
+**As an** internet user (a submitter or a data subejct), when creating a Data rights request (Global data rights request capture service)
+
+**I want to** add the address of the organization I am sending my request to, that doesn't use blindnet devkit/Privaform, and be able to easily access it next time
+
+**because** I or other users might need the address of that organization to send Data rights request in the future
+
+### US-07-1-06
+
+**As an** internet user (a submitter or a data subejct), when creating a Data rights request
+
+**I want** be able to make any of the requests mentionned in the [requests list of this document](https://github.com/blindnet-io/product-management/blob/devkit-schemas/refs/schemas/RFC-rights-request-interoperability-format.md)
+
+**because** this list of requests covers my data rights needs
+
+### US-07-1-07
+
+**As an** internet user (a submitter or a data subejct), when creating a Data rights request
+
+**I want to** send several requests at the same time
+
+**because** I want to exercise several data rights on the data an organization has one me
+
+### US-07-1-08
+
+**As an** internet user (a submitter or a data subejct), when creating a Data rights request
+
+**I want to** modify my address and to modify a field from an allergy record 
+
+**because** because I just moved and also dicovered I have a new allergy
+
+## 07-2 Organization receiving Data rights request (Global data rights request capture service)
+
+### US-07-2-01
+
+**As an** organization (not using blindnet devkit/Privateform) receiving Data rights request 
+
+**I want to** know the request I received has been sent by blindnet Data rights request capture service and be informed about Privateform
+
+**because** I might be interested in the service Privateform offers
+
+### US-07-2-02
+
+**As an** organization (not using blindnet devkit/Privateform) receiving Data rights request 
+
+**I want to** be able to recognize the user who made the request in my system
+
+**because** so I can easily identify their data
+
+### US-07-2-03
+
+**As an** organization, using blindnet devkit/Privateform, receiving Data rights request 
+
+**I want to** receive the request in the Data rights request section of my system/my Privatefrom account
+
+**because** it is more convenient for me to have all my Data rights requests centralised
+
+### US-07-2-04
+
+**As an** organization (using or not using blindnet devkit/Privateform) receiving Data rights request
+
+**I want to** transfer the request to another organization if the request concerns data I transfered to another organization
```

Comment:
What do you mean by someone else ? Another organization ?

Reply:
I updated the **US-07-2-04** :

**As an** organization (using or not using blindnet devkit) receiving Privacy request

**I want to** transfer the request to another organization if the request concerns data I transfered to another organization or data I got from another organization

**because** I need to be compliant, if a user wants their data deleted, I have to inform the other company I transfered the user's data to or I got the user's data from to also delete the data

Reply:
LGTM